content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
 ```
 
-> Note:
+> Catatan:
 >
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
+> *Web Components* sering kali memiliki API yang bersifat imperatif. Contohnya, *Web Component* video mungkin memiliki fungsi `play()` dan `pause()`. Untuk mengakses API yang imperatif dari sebuah *Web Component*, Anda perlu menggunakan referensi untuk berinteraksi dengan simpul DOM secara langsung. Jika Anda menggunakan *Web Components* pihak ketiga, solusi terbaik adalah dengan menulis komponen React yang membungkus *Web Component* Anda.
```

Comment:
```suggestion
> *Web Components* sering kali memiliki API yang bersifat imperatif. Contohnya, *Web Component* video mungkin memiliki fungsi `play()` dan `pause()`. Untuk mengakses API yang imperatif dari sebuah *Web Component*, Anda perlu menggunakan `ref` untuk berinteraksi dengan simpul DOM secara langsung. Jika Anda menggunakan *Web Components* pihak ketiga, solusi terbaik adalah dengan menulis komponen React yang membungkus *Web Component* Anda.
```

Since `ref` that mentioned here is the `ref` API from React 😄